scylla-gdb.py

```diff
@@ -3504,7 +3504,12 @@ def find_sstables_attached_to_tables():
 
 def find_sstables():
     """A generator which yields pointers to all live sstable objects on current shard."""
-    for sst in intrusive_list(gdb.parse_and_eval('sstables::tracker._sstables'), link='_tracker_link'):
+    db = find_db(current_shard())
+    user_sstables_manager = std_unique_ptr(db["_user_sstables_manager"]).get()
+    system_sstables_manager = std_unique_ptr(db["_system_sstables_manager"]).get()
+    for sst in intrusive_list(user_sstables_manager["_active"], link='_manager_link'):
+        yield sst.address
+    for sst in intrusive_list(system_sstables_manager["_active"], link='_manager_link'):
```

Comment:
Maybe also include `_undergoing_close`, since those sstables still exist in the system (and can take up memory, for example).

Reply:
@mikolajsieluzycki I was about to commit your code as is, but maybe you can do this too and force-push the new version (you don't need to create a new pull request - just force-push the branch again under the same name).

Reply:
@denesb why is that? Did we not have sstable_manager in one of the live versions?
To solve this scylla-gdb.py would need to try sstable_manager first, and if it doesn't exist, fall back to use sstable_tracker. Should be fairly easy to do.

Reply:
Right, I forgot that we have `sstable_manager` for some time. Maybe we don't even need that check, we (@mikolajsieluzycki ) just need to check that all live versions have it. If they do, we can make the change as here.